tfjs-backend-webgpu/karma.conf.js

```diff
@@ -71,7 +69,7 @@ module.exports = function(config) {
     customLaunchers: {
       chrome_webgpu: {
         base: 'Chrome',
-        flags: ['--enable-unsafe-webgpu'],
+        flags: ['--enable-unsafe-webgpu', '--disable-dawn-robustness'],
```

Comment:
I assume this change wasn't included with the 20% number?

Reply:
Yes, you are right. Split it into another patch #4238 in case causing confusion. Thanks.